.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+python:
+  - 2.7
+before_install:
+  - export DISPLAY=:99
+  - sh -e /etc/init.d/xvfb start
+install:
+  - pip install -r test-requirements.txt
+  - pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements/base.txt
+  - pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements/test.txt
+  - pip install -r $VIRTUAL_ENV/src/xblock/requirements.txt
```

Comment:
@nedbat Are you saying we should use `cd $VIRTUAL_ENV/src/xblock-sdk/ && make pip` instead of the commands that we are using now (which are the same as what's in the Makefile)? Or something else?